python/cudf/cudf/_lib/cpp/stream_compaction.pxd

```diff
@@ -33,13 +34,14 @@ cdef extern from "cudf/stream_compaction.hpp" namespace "cudf" \
         column_view boolean_mask
     ) except +
 
-    cdef unique_ptr[table] unique(
-        table_view source_table,
-        vector[size_type] keys,
-        duplicate_keep_option keep,
-        null_equality nulls_equal) except +
-
     cdef size_type distinct_count(
         column_view source_table,
         null_policy null_handling,
         nan_policy nan_handling) except +
+
+    cdef unique_ptr[table] stable_distinct(
+        table_view input,
+        vector[size_type] keys,
+        duplicate_keep_option keep,
+        null_equality nulls_equal,
+    ) except +
```

Comment:
```suggestion
        null_equality nulls_equal) except +
```

Reply:
There's no preference here from our formatting checkers, both pass. We're inconsistent about this throughout the Cython code and I'd prefer to keep the current style because new parameters introduce a more minimal diff.